packages/cli/src/oclif/commands/promote.js

```diff
@@ -35,8 +45,21 @@ class PromoteCommand extends BaseCommand {
     const assumeYes = 'yes' in this.flags;
 
     let shouldContinue;
-    const changelog = await getVersionChangelog(version);
-    if (changelog) {
+    const { changelog, appMetadata, issueMetadata } = await getVersionChangelog(
+      version
+    );
+    if (!changelog) {
+      this.error(`${colors.yellow(
+        'Warning!'
+      )} Changelog not found. Please create a CHANGELOG.md file in a format similar to ${colors.cyan(
+        'https://gist.github.com/xavdid/b9ede3565f1188ce339292acc29612b2'
```

Comment:
For [PDE-3820](https://zapierorg.atlassian.net/browse/PDE-3820), can we hard code a self-contained example in the code so we can remove this personal gist?

Reply:
Sure. This example is enormous so I'm just going to copy a subset, esp. since it'll now be printed inline vs. a link